js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -57,6 +57,10 @@ const MediaLightbox = new Module.Class({
     },
 
     _on_close: function () {
+        if (this.lightbox_widget instanceof EosKnowledgePrivate.MediaBin) {
```

Comment:
In the future consider `if ('stop' in this.lightbox_widget)`, I think it's more Javascript-y, but no big deal